nerdlandbot/commands/kerk.py

```diff
@@ -0,0 +1,30 @@
+import typing
+import asyncio
+import discord
+from discord.ext import commands
+from nerdlandbot.translations.Translations import get_text as translate
+from nerdlandbot.helpers.TranslationHelper import get_culture_from_context as culture
+from nerdlandbot.helpers.parser import parse_channel
+
+from datetime import datetime
+
+class Kerk(commands.Cog, name="kerk"):
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="16ukerk", hidden=True)
+    async def cmd_kerk16(self, ctx: commands.Context, mention:typing.Optional[str] = None, *, message:typing.Optional[str] = None):
+        #check if message in lounge
+        if ctx.channel.name == ("lounge"):
```

Comment:
Feature should not be limited to lounge

Reply:
I would limit it to the lounge (or to botplayground) to avoid clutter, but identify the channel by ID instead of by name

Reply:
It's not a good practice to add constraints to external factors.
What if one of these channels is renamed?
If we want to limit this functionality to certain channels, we could, but i would define a channel group that allows for the given functionality and make it editable either through discord, or a config file, so we at least won't need a code change